### DISCUSSION TITLE
Cumulative Displacement

### DISCUSSION THREAD

**Discussion by galuhqfs** [gmtsar/gmtsar#438] [Q&A] [answered]
Hi,

I ran the sbas process and the result was disp_*.grd and vel.grd. I have a total 28 displacement for the time-series result. I need a file called cumulative displacement that given by all the displacement, but it doesn't appears on sbas result. How can I get the cumulative displacement from my time-series displacement?

Thank you

**Comment by Xiaohua-Eric-Xu** [comment, marked as answer]
Hi @galuhqfs, the output from sbas should be cumulative displacement.

**Reply by galuhqfs** [reply]
Hi Eric, thanks for the answer. But, I have an issue on my vel.grd result. All the process of making sbas has done. The value of Z range on my vel.grd showed a high displacement (uplift and subsidence) and only has blue color around the region of area. For your information, I did the unwrapping process using `snaphu_interp.csh 0.10 40 /minRange/maxRange/minAzimuth/maxAzimuth`. So I did nothing for the masking process. And for the sbas process, I used `sbas intf.tab scene.tab 33 28 7606 4680 -wavelength 0.236 -incidence 37.000 -smooth 5.0 -range 866000 -rms -dem`.

Is the problem of my disp caused by the unwrap result without masking? And how is the solution? Thank you.
I'll attach my scene.tab and intf.tab file below.
[intf.txt](https://github.com/gmtsar/gmtsar/files/9641140/intf.txt)
[scene.txt](https://github.com/gmtsar/gmtsar/files/9641142/scene.txt)

**Reply by Xiaohua-Eric-Xu** [reply]
Could it be that you did not reference your data to a pixel? e.g. setting up a zero point in your dataset? This can be done with 
```
foreach line (`cat intflist`)
    cd $line
    gmt grdcut unwrap.grd -Rx1/x2/y1/y2 -Gtmp.grd
    set a = `gmt grdinfo tmp.grd -L2 -C | awk '{print $12}'`
    gmt grdmath unwrap.grd $a SUB = unwrap_pin.grd
    cd ..
end
```
Here x1/x2/y1/y2 denotes an area that you'd like to pin to zero.

**Reply by galuhqfs** [reply]
I don't understand about x1/x2/y1/y2 part.

So we need to point out an area and make it as 0,0 based on the unwrap.grd info. Based on script that you given to me (-Rx1/x2/y1/y2), there are 2 point that has to be defined as (0,0) on the area. Is it a free points or there is an certain arrangement while making the points?

Is the points (x1/x2/y1/y2),  in radar coordinates or long,lat coordinates?
![WhatsApp Image 2022-09-27 at 19 06 00 (1)](https://user-images.githubusercontent.com/106435027/192522349-cbdd916e-dcbb-4819-a257-e2eca6e460b7.jpeg)

**Reply by Xiaohua-Eric-Xu** [reply]
You could either select a point or an area, though a point (pixel) is essentially an area in a radar image. There are suggestions like "select a area that is quiet from atmospheric noise", or where "GNSS data shows a steady rate and the coherence is good". 

And yes, in radar coordinates if you are running sbas over data in radar coordinates, and geo-coordinates if you are running sbas in geo-coordinates.

**Reply by Xiaohua-Eric-Xu** [reply]
If you are doing a point, then using grdtrack might be easier and faster.
